src/views/apply/form.html

```diff
@@ -1,14 +1,25 @@
 {% extends 'default.html' %}
-{% set title = 'Apply for Hack Cambridge' %}
-{% block pageHeading %}Apply{% endblock %}
+{% set title = 'Application' %}
+{% set footer = true %}
+{% block pageHeading %}Application{% endblock %}
 {% block pageContent %}
-<div class="content-wrapper-center">
-  <h2>Help us get to know you a little better, {{ user.firstName }}</h2>
+<div class="one unit row">
+    Hey {{ user.firstName }}, thanks for choosing to apply.
```

Comment:
Maybe just "thanks for your application"?

Reply:
The problem is the actual application happens when the person clicks submit, what about 'thanks for applying to Hack Cambridge'?